clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/input/kafka/VeniceKafkaInputMapper.java

```diff
@@ -24,12 +28,23 @@ protected AbstractVeniceRecordReader<BytesWritable, KafkaInputMapperValue> getRe
     throw new UnsupportedOperationException();
   }
 
+  @Override
+  protected Optional<AbstractVeniceFilter<KafkaInputMapperValue>> getFilter(final VeniceProperties props) {
+    long ttlInHours = props.getLong(VenicePushJob.REPUSH_TTL_IN_HOURS_OVERRIDE, -1);
+    if (ttlInHours != -1) {
+      VeniceKafkaInputTTLFilter filter = new VeniceKafkaInputTTLFilter(props);
+      return Optional.of(filter);
+    }
+    return Optional.empty();
```

Comment:
Is the output of this method going to be used in lambada/streams?

Reply:
No.

Reply:
If not let's not use optional then